### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by andrewm4894**
##### Summary

Draft PR to update our main README in a few places where needed.

discussion in here: https://github.com/netdata/website/issues/95

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +20,18 @@
 
 <img src="https://user-images.githubusercontent.com/1153921/95269366-1b814680-07ee-11eb-8ff4-c1b0b8758499.png" alt="---" style="max-width: 100%;" />
 
-Netdata's **distributed, real-time monitoring Agent** collects thousands of metrics from systems, hardware, containers,
-and applications with zero configuration. It runs permanently on all your physical/virtual servers, containers, cloud
-deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any
-preparation.
+Netdata is distributed, real-time, performance and health monitoring platform for systems, hardware, containers and applications with zero configuration needed. It runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any preparation.
```
Comment (andrewm4894) [inline README.md:23]:
```suggestion
Netdata is a distributed, real-time, performance and health monitoring platform for systems, hardware, containers and applications with zero configuration needed. It runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any preparation.
```

File: README.md [hunk 2]
```diff
@@ -20,10 +20,18 @@
 
 <img src="https://user-images.githubusercontent.com/1153921/95269366-1b814680-07ee-11eb-8ff4-c1b0b8758499.png" alt="---" style="max-width: 100%;" />
 
-Netdata's **distributed, real-time monitoring Agent** collects thousands of metrics from systems, hardware, containers,
```
Comment (andrewm4894) [inline README.md:23]: kinda like the "thousands of metrics" phrase as is a bit of a differentiator tbh then just basic standard metrics from some other agents.

File: README.md [hunk 3]
```diff
@@ -20,10 +20,18 @@
...
-and applications with zero configuration. It runs permanently on all your physical/virtual servers, containers, cloud
-deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any
-preparation.
+Netdata is a distributed, real-time, performance and health monitoring platform for systems, hardware, containers and applications with zero configuration needed. It runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any preparation.
```
Comment (andrewm4894) [inline README.md:23]:
```suggestion
Netdata is a distributed, real-time, performance and health monitoring platform for systems, hardware, containers and applications, collecting thousands of useful metrics with zero configuration needed. It runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any preparation.
```

File: README.md [hunk 4]
```diff
@@ -20,10 +20,18 @@
...
+
+Netdata [Agent](https://github.com/netdata/netdata) is an enormously powerful, **Open-Sourced**, **Single Node** health monitoring and performance troubleshooting tool.
+It provides you an ability to automatically identify processes, collect and store metrics locally and even more - vizualize all metrics without any chart configuration (of course you can tweak it later on if you need).
+
+[Netdata.cloud](https://www.netdata.cloud) is a hosted web interface that gives you **Free**, real-time visibility into your **Entire Infrastructure** with secure access to your Netdata Agents. It provides an ability to automatically route your requests to the most relevant Netdata Agents to get the metric data, based on the stored metadata (Agents topology, what metrics are collected on specific Agents as well as the retention information for each metric).
```
Comment (andrewm4894) [inline README.md:28]:
```suggestion
[Netdata Cloud](https://www.netdata.cloud) is a hosted web interface that gives you **Free**, real-time visibility into your **Entire Infrastructure** with secure access to your Netdata Agents. It provides an ability to automatically route your requests to the most relevant Netdata Agents to get the metric data, based on the stored metadata (Agents topology, what metrics are collected on specific Agents as well as the retention information for each metric). 
```

File: README.md [hunk 5]
```diff
@@ -20,10 +20,18 @@
...
+
+Netdata [Agent](https://github.com/netdata/netdata) is an enormously powerful, **Open-Sourced**, **Single Node** health monitoring and performance troubleshooting tool.
+It provides you an ability to automatically identify processes, collect and store metrics locally and even more - vizualize all metrics without any chart configuration (of course you can tweak it later on if you need).
+
+[Netdata.cloud](https://www.netdata.cloud) is a hosted web interface that gives you **Free**, real-time visibility into your **Entire Infrastructure** with secure access to your Netdata Agents. It provides an ability to automatically route your requests to the most relevant Netdata Agents to get the metric data, based on the stored metadata (Agents topology, what metrics are collected on specific Agents as well as the retention information for each metric). 
+
+It gives you some extra features, like [Metric Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations), [Anomaly Advisor](https://learn.netdata.cloud/docs/cloud/insights/anomaly-advisor) and much more.
```
Comment (andrewm4894) [inline README.md:30]:
```suggestion
It gives you some extra features, like [Metric Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations), [Anomaly Advisor](https://learn.netdata.cloud/docs/cloud/insights/anomaly-advisor) and much more. 

Try it for yourself right now by checking out our Netdata Cloud [demo space](https://app.netdata.cloud/spaces/netdata-demo/rooms/all-nodes/overview) (No sign up or login needed).
```

File: README.md [hunk 6]
```diff
@@ -20,10 +20,18 @@
...
+It provides you an ability to automatically identify processes, collect and store metrics locally and even more - vizualize all metrics without any chart configuration (of course you can tweak it later on if you need).
+
+[Netdata.cloud](https://www.netdata.cloud) is a hosted web interface that gives you **Free**, real-time visibility into your **Entire Infrastructure** with secure access to your Netdata Agents. It provides an ability to automatically route your requests to the most relevant Netdata Agents to get the metric data, based on the stored metadata (Agents topology, what metrics are collected on specific Agents as well as the retention information for each metric). 
+
+It gives you some extra features, like [Metric Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations), [Anomaly Advisor](https://learn.netdata.cloud/docs/cloud/insights/anomaly-advisor) and much more. 
+
+The Netdata purpose is to help more people troubleshoot ever more complex IT infrastructures, this is why our **free** [community plan](https://www.netdata.cloud/pricing) gives you ability to monitor unlimited number of Nodes, Containers and Metrics (custom or built-in).
```
Comment (andrewm4894) [inline README.md:32]:
```suggestion
The Netdata mission is to help more people troubleshoot ever more complex IT infrastructures, this is why our **free** [community plan](https://www.netdata.cloud/pricing) gives you ability to monitor unlimited number of Nodes, Containers and Metrics (custom or built-in).
```

File: README.md [hunk 7]
```diff
@@ -110,6 +118,24 @@ Netdata works with tons of applications, notifications platforms, and other time
   <a href="https://hub.docker.com/r/netdata/netdata"><img src="https://registry.my-netdata.io/api/v1/badge.svg?chart=dockerhub.pulls_sum&divide=1000&precision=1&units=k&label=docker+hub+pulls&after=-86400&group=incremental-sum&label=docker%20hub%20pulls%20today&options=unaligned&v44" alt="Docker Hub pulls today"></a>
 </p>
 
+### Infrastructure view
+
+Due to the distributed nature of the Netdata ecosystem, it is recommended to setup not only not one Netdata Agent on your production system, but also an additional Netdata Agent acting as a [Parent](https://learn.netdata.cloud/docs/agent/streaming). A local Netdata Agent (child), without any database or alarms, collects metrics and sends them to another Netdata Agent (parent). The same parent can collect data for any number of child nodes and serves as a centralized health check engine for each child by triggering alerts on their behalf.
```
Comment (andrewm4894) [inline README.md:123]:
```suggestion
Due to the distributed nature of the Netdata ecosystem, it is recommended to setup not only one Netdata Agent on your production system, but also an additional Netdata Agent acting as a [Parent](https://learn.netdata.cloud/docs/agent/streaming). A local Netdata Agent (child), without any database or alarms, collects metrics and sends them to another Netdata Agent (parent). The same parent can collect data for any number of child nodes and serves as a centralized health check engine for each child by triggering alerts on their behalf.
```

File: README.md [hunk 8]
```diff
@@ -110,6 +118,24 @@ Netdata works with tons of applications, notifications platforms, and other time
...
+
+![Netdata Cloud](https://user-images.githubusercontent.com/423236/205926887-43024984-6d38-46ad-96cb-d0c388117c6d.png)
+
+Get started by [signing in](https://app.netdata.cloud/?utm_source=website&utm_content=top_navigation_sign_up) to Netdata.cloud and follow the setup guide.
```
Comment (andrewm4894) [inline README.md:127]:
```suggestion
Get started by [signing in](https://app.netdata.cloud/?utm_source=website&utm_content=top_navigation_sign_up) to Netdata Cloud and follow the setup guide.
```

Comment (amalkov) [reply README.md:127]: I deliberately put netdata.cloud to promote the website

**Comment by amalkov** [comment]
@andrewm4894 @hugovalente-pm Thank you guys for all your suggestions, agree with everything, just one minor comment: I deliberately used Netdata.cloud to promote the website, what do you think about it?

**Comment by andrewm4894** [comment]
> @andrewm4894 @hugovalente-pm Thank you guys for all your suggestions, agree with everything, just one minor comment: I deliberately used Netdata.cloud to promote the website, what do you think about it?

yeah i dont know feels kinda strange to me or just "looks off" for some reason. I think a blue link "Netdata Cloud" makes enough sense. vwery subjective though and just my opinion - maybe we see what others think.

File: README.md (hunk 8 above)
Comment (hugovalente-pm) [reply README.md:127]: I think we could to like @andrewm4894 suggests, have [Netdata Cloud](https://www.netdata.cloud/)

File: README.md (hunk 5 above)
Comment (sashwathn) [inline README.md:30]: It gives you some extra features to troubleshoot your infrastructure holistically, like [Metric Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations), [Anomaly Advisor](https://learn.netdata.cloud/docs/cloud/insights/anomaly-advisor), [anomaly rates on every chart](https://blog.netdata.cloud/anomaly-rate-in-every-chart/) and much more.

File: README.md [hunk 9]
```diff
@@ -20,10 +20,18 @@
...
+
+It gives you some extra features, like [Metric Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations), [Anomaly Advisor](https://learn.netdata.cloud/docs/cloud/insights/anomaly-advisor) and much more. 
+
+The Netdata purpose is to help more people troubleshoot ever more complex IT infrastructures, this is why our **free** [community plan](https://www.netdata.cloud/pricing) gives you ability to monitor unlimited number of Nodes, Containers and Metrics (custom or built-in).
+
+Due to the distributed nature of Netdata, and to ensure high-availability of your monitoring system, please check our [Data Replication](https://www.netdata.cloud/blog/why-is-data-replication-important) recommendations to increase the data availability.
```
Comment (sashwathn) [inline README.md:34]: Due to the distributed nature of Netdata and to ensure high-availability of your monitoring system, please check our [Data Replication](https://www.netdata.cloud/blog/why-is-data-replication-important) recommendations to increase the data availability.

File: README.md (hunk 8 above)
Comment (sashwathn) [reply README.md:127]: I think Netdata Cloud sounds better too.

**Review by sashwathn — CHANGES_REQUESTED** [review]

**Comment by amalkov** [comment]
> > @andrewm4894 @hugovalente-pm Thank you guys for all your suggestions, agree with everything, just one minor comment: I deliberately used Netdata.cloud to promote the website, what do you think about it?
> 
> yeah i dont know feels kinda strange to me or just "looks off" for some reason. I think a blue link "Netdata Cloud" makes enough sense. very subjective though and just my opinion - maybe we see what others think.

@andrewm4894 agree with you. Thanks!

**Comment by amalkov** [comment]
guys, are we happy to merge?

**Comment by andrewm4894** [comment]
> guys, are we happy to merge?

Yep I say merge and sure we can just tweak any further over time bit by bit.

**Comment by hugovalente-pm** [comment]
> > guys, are we happy to merge?

> Yep I say merge and sure we can just tweak any further over time bit by bit.

I agree, I'll give my approval on this

**Review by hugovalente-pm — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (andrewm4894) [reply README.md:30]: can you make a commit suggestion as think this is blocking merge

File: README.md [hunk 10]
```diff
@@ -20,10 +20,18 @@
...
 
-Netdata's **distributed, real-time monitoring Agent** collects thousands of metrics from systems, hardware, containers,
-and applications with zero configuration. It runs permanently on all your physical/virtual servers, containers, cloud
-deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any
-preparation.
+Netdata is a distributed, real-time, performance and health monitoring platform for systems, hardware, containers and applications with zero configuration needed. It runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices, and is perfectly safe to install on your systems mid-incident without any preparation.
+
+Netdata [Agent](https://github.com/netdata/netdata) is an enormously powerful, **Open-Sourced**, **Single Node** health monitoring and performance troubleshooting tool.
+It provides you an ability to automatically identify processes, collect and store metrics locally and even more - vizualize all metrics without any chart configuration (of course you can tweak it later on if you need).
```
Comment (cakrit) [inline README.md:26]:
```suggestion
It gives you the ability to automatically identify processes, collect and store metrics locally and even more - visualize all metrics without any configuration (of course you can tweak it later on if you need).
```

**Comment by andrewm4894** [comment]
@sashwathn can you approve or add commit suggestions - think you would be blocking this since GH has you as "requested changes"

**Review by sashwathn — DISMISSED** [review]

**Comment by sashwathn** [comment]
@andrewm4894 : I have approved this pull request.

**Review by cakrit — APPROVED** [review]

**Pull request merged**